test/helpers.bash

```diff
@@ -104,7 +104,7 @@ if [ -e /usr/sbin/selinuxenabled ] && /usr/sbin/selinuxenabled; then
     filelabel=$(awk -F'"' '/^file.*=.*/ {print $2}' /etc/selinux/${SELINUXTYPE}/contexts/lxc_contexts)
     chcon -R ${filelabel} $TESTDIR
 fi
-LIBPOD_CNI_CONFIG="$TESTDIR/cni/net.d/"
+LIBPOD_CNI_CONFIG="/etc/cni/net.d/"
```

Comment:
Why are you installing directly into the system?  Shouldn't this stay in the TESTDIR?

Reply:
With papr and travis being so different, we need to commit to installing stuff to the system so tests can be done similarily or we will be forever chasing our tails -- imho

Reply:
I don't think we should be doing this, why can't we just install the two files into the $TESTDIR/cni/net.d from the helpers.bash script?  If we use the system path, we should not install to it, and just require that these packages be installed.  

I am fearful that we will override some defaults that a user is actually using.